framework/sandstone_ssl.cpp

```diff
@@ -41,7 +41,7 @@ void sandstone_ssl_init()
     };
 #else
     void *libcrypto;
-    libcrypto = dlopen("libcrypto.so." SHLIB_VERSION_NUMBER, RTLD_NOW);
+    libcrypto = dlopen("libcrypto.so", RTLD_NOW);
```

Comment:
This won't work because on most non-development systems, the .so symlink is not present. You need to keep the SHLIB_VERSION_NUMBER.